slides/erb/index.html

```diff
@@ -0,0 +1,348 @@
+  DONG 🔔
+  DONG 🔔
+  DONG 🔔
+  DONG 🔔
+  DONG 🔔
+
+  Good morning Bill</code></pre>
+          </section>
+        </section>
+
+        <section>
+          <section>
+            <h2>Separation of Concerns</h2>
+
+            <p>
+              We can separate our code into <b>different files</b>, allowing us to focus on our <b>application logic</b> in one file and our <b>presentation logic</b> in another
+            </p>
+          </section>
+
+          <section>
+            <h3>Application logic</h3>
+
+            <pre><code class="hljs ruby">name = "Bill"
+lucky_numbers = (1..50).to_a.sample(6)
+</code></pre>
+          </section>
+
+          <section>
+            <h3>Presentation logic</h3>
+
+            <p class="text-xl"><code>File: ./lucky_numbers.txt</code></p>
+            <pre><code class="hljs erb">Hi <%= name %> 👋
+
+Today your lucky numbers are:
+
+<% lucky_numbers.sort.each do |lucky_number| %>
+  * <%= lucky_number %>
+<% end %>
+
+Good luck! 🍀
+</code></pre>
+          </section>
+
+          <section>
+            <h3>Rendering</h3>
+
+            <pre><code class="hljs ruby">require "erb"
+template = File.read("./lucky_numbers.txt")
+ERB.new(template).result(binding)</code></pre>
+          </section>
+
+          <section>
+            <h3>Output</h3>
+
+            <pre><code class="hljs text">Hi Bill 👋
+
+Today your lucky numbers are:
+
+  * 10
+  * 12
+  * 23
+  * 33
+  * 39
+  * 45
+
+Good luck! 🍀</code></pre>
+          </section>
+        </section>
+
+        <section>
+          <h2>Reusability</h2>
+
+          <p>
+            ERB templates can be reused across different parts of our application.
+          </p>
+        </section>
+
+        <section>
+          <section>
+            <h2>HTML + ERB = ❤️</h2>
+          </section>
+
+
+          <section>
+            <p>Let's create an HTML template</p>
+
+            <p class="text-xl"><code>File ./chores.html</code></p>
+            <pre><code class="hljs html">&lt;h1&gt;Hi <%= username %> 👋&lt;/h1&gt;
+
+&lt;p&gt;Don't forget to do following chores today:&lt;/p&gt;
+
+&lt;ul&gt;
+  <% chores.each do |chore| %>
+    &lt;li&gt;<%= chore %>&lt;/li&gt;
+  <% end %>
+&lt;/ul&gt;</code></pre>
+            <p class="text-lg">Remember, this is the <b>presentation logic</b></p>
+          </section>
+
+          <section>
+            <p>Application logic</p>
+
+            <pre><code class="hljs ruby">username = "Bill"
+
+chores = [
+  "Call mom ☎️",
+  "Clean the bath 🫧",
+  "Walk the dog 🐕",
+  "Learn Spanish 🇪🇸"
+]</code></pre>
+          </section>
+
+
+          <section>
+            <p>Rendering</p>
+
+            <pre><code class="hljs ruby">require "erb"
+template = File.read("./chores.html")
+ERB.new(template).result(binding)</code></pre>
+          </section>
+
+          <section>
+            <p>HTML Output</p>
+            <pre><code class="hljs html">&lt;h1&gt;Hi Bill 👋&lt;/h1&gt;
+
+&lt;p&gt;Don't forget to do following chores today:&lt;/p&gt;
```

Comment:
```suggestion
&lt;p&gt;Don't forget to do the following chores today:&lt;/p&gt;
```

not that you also have to fix the screenshot ;)